### DISCUSSION TITLE
Contribute to Finos Common Domain Model

### DISCUSSION THREAD

**Discussion by Teyiowuawi** [finos/common-domain-model#3173] [General]
I work with a team of modellers and we would like to contribute to the FINOS common domain model open source project. How could this be done and what are the required approvals needed?

**Comment by chrisisla** [comment]
Hi @Teyiowuawi 

Great to have you aboard! 

We have details of how to contribute on the landing page here: 
https://github.com/finos/common-domain-model?tab=readme-ov-file#contributing

Basically, in order to contribute changes to the model yourself you will need to have an active Individual Contributor License Agreement (ICLA) or be covered by a Corporate Contribution License Agreement (CCLA). You can always join any of the CDM working groups to get involved too (these are all listed on the FINOS Community Calendar and their Agendas and Minutes can be found in the Issues list)

Thanks!
